templates/confluence.cfg.xml.erb

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<confluence-configuration>
+  <setupStep>setupdbchoice-start</setupStep>
+  <setupType>custom</setupType>
+  <buildNumber>0</buildNumber>
+  <properties>
+    <property name="atlassian.license.message"><%= scope.lookupvar('confluence::license') -%></property>
+    <property name="confluence.setup.server.id"><%= scope.lookupvar('confluence::server_id') -%></property>
+    <property name="confluence.webapp.context.path"></property>
+    <property name="lucene.index.dir">${localHome}/index</property>
+    <property name="synchrony.btf">true</property>
+    <property name="synchrony.encryption.disabled">true</property>
+    <property name="synchrony.proxy.enabled">true</property>
```

Comment:
Should these be made configurable? Are these enough to begin with?

In our Confluence setup. This file contains the database config (though I suppose it could be in server.xml as well) and other things like the jwt private key. what happens when those are left out?

If this file changes, will puppet override it again?

Reply:
@n2ygk these are yours, you might better answer these questions

Reply:
I will say that we set up an `.epp` template here that's more comprehensive
https://gist.github.com/ronech/e7fd320f4c13eb614a611083fe741cff

Reply:
@ronech I can take a look.  

Can you see why the travis build failed for Puppet 4? Are you using some Puppet 5+ features? I couldn't really tell from the build logs.

Reply:
on it

Reply:
talked it over with a coworker and i'm gonna replace the `.erb` with our `.epp`

Reply:
Got too tricky and didn't want to refactor too much.  I believe i've fixed the Strings issue and added a couple of things.  Our `epp` was built on us puppetizing our extant Confluence instance, which has some assumptions built in.

Reply:
Travis is happy.  Are you satisfied, @oranenj?